packages/ez-react/src/docs/3_yourfirstchart.stories.mdx

```diff
@@ -0,0 +1,116 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Get Started/Your First Chart" parameters={{ options: { isToolshown: false }, id: 3 }} />
+
+# Your First Chart
+
+Now that you installed **EazyChart**, you will have the ability to use it and add a chart in your **React** or **Vue** web project. 
+
+EazyChart provides a set of ready-to-use charts that you can fully customize. In the example below, you will be adding a "Bar Chart" to a React project. First, you need to import the **BarChart** component from the library.
+
+```js
+// Code for importing the bar chart component
+import { BarChart } from 'eazychart-react'
+import 'eazychart-css'
+```
+
+The above code will import the Bar chart to where you want it to show. 
+So use it to import the chart inside the parent component.
+
+A quick note, to get the appropriate styles for your chart, you’ll have to import the `eazychart-css` package. 
+
+> **NOTE**  `!` 
+> **You can import it once into your app main file, and it won't be systematic with every chart import.**
+
+Cool, now that you imported the Bar chart from the library, you are ready to use it in your project. 
+
+Hang on there, you still have a couple of things to do. You need to define the data that the chart should display: 
+
+```js
+const data=[
+    { country: 'Italy', population: 65 },
+    { country: 'Spain', population: 46 },
+    { country: 'Australia', population: 25 },
+];
+```
+Awesome. Now you’ll need to configure the chart. To do so, we will need to create an object that will contain all the props we need to define.
+
+As an example. In order to display a Bar chart into your project, you will need to specify : 
+
+* The data that you will be displaying (data).
+* The axes that you will use to represent the data on. Basic charts typically have two axes that are used to plot data in two dimensions (X Axis and Y Axis).
+
+```js
+const props = { data : data, 
+  xAxis: {
+    domainKey: 'population',
+    title: 'Population',
+    nice: 2,
+  },
+  yAxis: {
+    domainKey: 'country',
+    title: 'Country',
+  } 
+};
+```
+The **xAxis** is for defining the horizontal axis or the x-axis that will display the values in the data (numerical values). **yAxis** is for defining the vertical axis or y-axis that will display the value part of the data (categorical values). 
+
+You can specify other chart properties (please refer to documentation about **Bar Chart** for more details). 
+
+For every axis, we will need to define a set of properties, mainly the axis direction, the domain key and the title. For the x-axis, we are defining a property called **‘nice’** that will add a margin so that the least value can be displayed. 
+
+![Example of Bar Chart](gif)
+
+You can learn more about more properties in the Axis section.
+
+Cool, now that everything is set up in place, let’s write the JSX code to display the chart using the props that we defined earlier.  
+
+Your code should look like this: 
+
+```jsx
```

Comment:
```suggestion
```react
```

Reply:
I think react doesn't work
this is the output when I put jsx: 
![image](https://user-images.githubusercontent.com/48720958/194367648-b9f0fc8e-e50a-43c9-8f2d-6bd46c907b11.png)

And this is the output when I put react: 
![image](https://user-images.githubusercontent.com/48720958/194367767-cabd17a4-bc8f-40f0-b498-ba2e1170e298.png)